app/dash_app.py

```diff
@@ -1,383 +1,397 @@
-            'LavaFork_3M20': [56.42, -130.85],
-            'LavaFork_3M21': [56.42, -130.85],
-            'LavaFork_3M29': [56.42, -130.85],
-            'LavaFork_3M31': [56.42, -130.85],
-            'LavaFork_3M41': [56.42, -130.85]}
-
-# Coherence Pair Data
-dfCohFull = pd.read_csv('Data/Meager/5M3/CoherenceMatrix.csv')
-
-fig = px.imshow(np.rot90(np.fliplr(dfCohFull['Average Coherence'].to_numpy().reshape(len(dfCohFull['Reference Date'].unique()),
-                                                                                     len(dfCohFull['Reference Date'].unique())))),
-                x=dfCohFull['Reference Date'].unique(),
-                y=dfCohFull['Pair Date'].unique(),
-                color_continuous_scale='RdBu_r'
-                )
-fig.update_yaxes(autorange=True)
-
-app.layout = html.Div(id='parent', children=[
-    html.Div(style={'width': '5%', 'display': 'inline-block'}),
-    html.Img(src=app.get_asset_url(
-                'Seal_of_the_Geological_Survey_of_Canada.png'),
-             style={'width': '10%',
-                    'display': 'inline-block'}),
-    html.Div(style={'width': '5%',
-                    'display': 'inline-block'}),
-    html.H1(id='H1',
-            children='Volcano InSAR Interpretation Workbench',
-            style={'textAlign': 'center',
-                   'marginTop': 40,
-                   'marginBottom': 40,
-                   'display': 'inline-block'}),
-    html.Div(style={'height': '10px'}),
-    html.Div(style={'width': '5%',
-                    'display': 'inline-block'}),
-
-    html.Div(
-        dcc.Dropdown(stackList, stackList[1], id='site-dropdown'),
-        style={'width': '35%',
-               'display': 'inline-block',
-               'color': 'black'}),
-
-    html.Div(style={'width': '5%',
-                    'display': 'inline-block'}),
-
-    html.Div([
-        dcc.Graph(id='graph', figure=fig,
-                  style={'position': 'absolute',
-                         'width': '25%',
-                         'height': '45%',
-                         'margin-left': '69.5%',
-                         'margin-top': '0.5%',
-                         'zIndex': 2}
-                         ),
-        dl.Map([dl.TileLayer(),
-                dl.LayersControl(
-                    dl.BaseLayer(
-                            dl.TileLayer(
-                                # Basemaps
-                                url='https://basemap.nationalmap.gov/arcgis/rest/services/USGSTopo/MapServer/tile/{z}/{y}/{x}',
-                                attribution='Tiles courtesy of the <a href="https://usgs.gov/">U.S. Geological Survey</a>'
+    """Parse configuration from supplied file."""
+    config_obj = configparser.ConfigParser()
+    config_obj.read(args)
+    return config_obj
+
+
+config = get_config_params('config.ini')
+GEOSERVER_ENDPOINT = config.get('geoserver', 'geoserverEndpoint')
+
+# TODO add support for some or all of the following parameters to config
```

Comment:
remove 'ToDo' and create issue

Reply:
Issue exists #11 and we can remove TODO when it is closed.